WordPress/wordpress-local-development.md

```diff
@@ -0,0 +1,95 @@
+{{{
+  "title": "Local Development with CenturyLink WordPress as a Service",
+  "date": "07-30-2015",
+  "author": "Matt Wittmann",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+Local Development with CenturyLink WordPress as a Service
```

Comment:
Didn't catch this before i merged this PR, but you don't need to add the title of the article in the body. The title is generated from what's in the front matter.